apps/www/_blog/2024-05-09-meetup-kahoot-alternative.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Kahoot alternative at GA week community meetup'
+description: How we built a Kahoot alternative for our Supabase meetup to boost up the community meetup.
+author: tyler_shukert
+image: oss-kahoot-alternative/thumbnail.png
+thumb: oss-kahoot-alternative/thumbnail.png
+launchweek: 11
+categories:
+  - launch-week
+tags:
+  - launch-week
+date: '2024-05-09'
+toc_depth: 2
+---
+
+Supabase GA week brought us many cool and exciting new features, but we also had so much fun worldwide through the community meetup. With the help of the amazing community leaders, we had meetups at over 30 different locations worldwide! Seeing all the fun pictures and videos of the community meetups was amazing!
+
+To make the meetups more engaging, we prepared an interactive quiz game that everyone could play. We were initially going to use Kahoot for this. Still, we thought it would be an excellent opportunity to showcase the power of Supabase and open source, so we built an open-source version of Kahoot with some pre-filled Supabase / tech questions for everyone to play at the meetup.
+
+## What we built for the meetup
+
+![Kahoot alternative host page](/images/blog/oss-kahoot-alternative/host-page.png)
+
+The Kahoot alternative that we built is a simplified version of Kahoot. Mainly having fewer features on the admin side of things, but as an individual player participating, the experience should be similar to the actual Kahoot.
+
+The game has two screens: the host screen and the player screen. The host can start a game by choosing one of the available quiz sets. Because we had meetups from around the globe, we created the same 10-question quiz sets in 4 different languages.
+
+![Dashboard](/images/blog/oss-kahoot-alternative/dashboard.png)
+
+Once the host clicks the button to start a game, a lobby screen with a QR code appears. The players can scan the QR codes with their phones, name themselves, and enter the game.
+
+The host can start the quiz game once everyone in the room has entered the lobby. The players are shown the question first and then the choices five seconds later for each question. They answer the quiz on their phones, and the sooner they answer, the more points they get.
+
+Once all the questions are answered, the points are summed up, and the leaderboard is shown on the screen.
+
+Actual leaderboard from the Tokyo meetup.
+
+![Leaderboard from Tokyo](/images/blog/oss-kahoot-alternative/results.png)
+
+## Building the Kahoot alternative app
+
+I will not go into every implementation detail, as it would be long, but you can find the GitHub repository with the full code [here](https://github.com/supabase-community/kahoot-alternative). The app is a Next.js app without using any server-side rendering, and it utilizes three core Supabase features: auth, database, and Real-Time.
```

Comment:
```suggestion
I will not go into every implementation detail, as it would be long, but you can find the GitHub repository with the full code [here](https://github.com/supabase-community/kahoot-alternative). The app is a Next.js app without using any server-side rendering, and it utilizes three core Supabase features: auth, database, and realtime.
```